WTWSMS/events/VIET_Jewish_Flavor_Events.txt

```diff
@@ -3,7 +3,7 @@
 
 #FLAVOR EVENTS
 
-#A bigoted courtier questions why we must have Jews in the court
+#A courtier questions why we must have Jews in the court
```

Comment:
I'm not going to make a whole debate out of this, I just thought your reaction on "soapbox" commentary to be funny.